$OpenBSD: patch-src_3rdparty_webkit_JavaScriptCore_runtime_JSValue_h,v 1.1 2010/06/15 21:27:57 landry Exp $
Fix build on sparc64, from netbsd x11/qt4-libs/patches/path-az
--- src/3rdparty/webkit/JavaScriptCore/runtime/JSValue.h.orig	Thu Jun 10 13:38:28 2010
+++ src/3rdparty/webkit/JavaScriptCore/runtime/JSValue.h	Thu Jun 10 13:38:54 2010
@@ -476,7 +476,7 @@ namespace JSC {
             u.asBits.tag = CellTag;
         else
             u.asBits.tag = EmptyValueTag;
-        u.asBits.payload = reinterpret_cast<int32_t>(ptr);
+        u.asBits.payload = reinterpret_cast<intptr_t>(ptr);
     }
 
     inline JSValue::JSValue(const JSCell* ptr)
@@ -485,7 +485,7 @@ namespace JSC {
             u.asBits.tag = CellTag;
         else
             u.asBits.tag = EmptyValueTag;
-        u.asBits.payload = reinterpret_cast<int32_t>(const_cast<JSCell*>(ptr));
+        u.asBits.payload = reinterpret_cast<intptr_t>(const_cast<JSCell*>(ptr));
     }
 
     inline JSValue::operator bool() const
